### DISCUSSION TITLE
Failed to start the Kernel.  Failed to start the Kernel 'Python 3.10.1 32-bit'.  View Jupyter log for further details. Kernel has not been started

### DISCUSSION THREAD

**Discussion by javeddildar** [AammarTufail/machinelearning_ka_chilla#6] [Q&A]
Failed to start the Kernel. 
Failed to start the Kernel 'Python 3.10.1 32-bit'. 
View Jupyter [log](command:jupyter.viewOutput) for further details. Kernel has not been started
![image](https://user-images.githubusercontent.com/78088008/158460375-5019cafc-68ff-4d20-8684-2652b5d140a5.png)
![Failedtostartkernal](https://user-images.githubusercontent.com/78088008/158460574-b053ae7f-0cc7-40ba-8711-77a579a23d9f.jpg)

**Comment by rasheed2063** [comment]
You may update your kernel and hopefully it will solve your issue
